holmes/core/prompt.py

```diff
@@ -89,10 +89,15 @@ def build_initial_ask_messages(
     """
     # Load and render system prompt internally
     system_prompt_template = "builtin://generic_ask.jinja2"
+    
+    # Load KAITO accuracy content and add it to system prompt additions
+    kaito_content = load_and_render_prompt("builtin://_kaito_accuracy.jinja2", {})
+    enhanced_system_prompt_additions = (system_prompt_additions or "") + f"\n\n{kaito_content}"
+    
     template_context = {
         "toolsets": tool_executor.toolsets,
         "runbooks_enabled": True if runbooks else False,
-        "system_prompt_additions": system_prompt_additions or "",
+        "system_prompt_additions": enhanced_system_prompt_additions,
```

Comment:
Instead of making kaito a system prompt addition, we can add it to the _general_instructions.jinja2. and if we want to toggle kaito we can use var like kaito_enabled like runbooks_enabled